server/models/cart.model.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const { Products } = require("./product.model");
+const cartSchema = new Schema({
+  products: [
+    {
+      productId: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: Products,
+      },
+      quantity: Number,
+    },
+  ],
+  userId: { type: mongoose.Schema.Types.ObjectId, ref: "User" },
```

Comment:
- You have already destructured `Schema` from mongoose so use that instead of `mongoose.Schema`.

Reply:
thanks for this...I didn't knew about this